### PR TITLE
Update LokiRingUnhealthy query to avoid false positive

### PR DESCRIPTION
**Pull request by marieroque**
towards https://github.com/giantswarm/giantswarm/issues/30469

### Checklist

- [x] Update CHANGELOG.md
- [x] Add [Unit tests](https://github.com/giantswarm/prometheus-rules/#testing)
- [ ] Follow [Alert structure](https://github.com/giantswarm/prometheus-rules/#how-alerts-are-structured)
- [ ] Consider [creating a dashboard](https://docs.giantswarm.io/getting-started/observability/grafana/custom-dashboards/) ([guidelines](https://intranet.giantswarm.io/docs/product/ux/guidelines/dashboards/)) (if it does not exist already) to help oncallers monitor the status of the issue.
- [ ] Request review from oncall area, as well as team (e.g: `oncall-kaas-cloud` GitHub group).

### CODE REVIEW

File: helm/prometheus-rules/templates/alerting-rules/loki.rules.yml [hunk 1]
```diff
@@ -73,7 +73,8 @@ spec:
         description: '{{`Loki pod {{ $labels.pod }} (namespace {{ $labels.namespace }}) sees {{ $value }} unhealthy ring members`}}'
         opsrecipe: loki/
       expr: |
-        sum (min_over_time(cortex_ring_members{state="Unhealthy"}[30m])) by (app, cluster_id, container, customer, installation, provider, pipeline, name, namespace, organization, pod) > 0
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/alerting-rules/loki.rules.yml:76]: Do WE want this to page only when app=Loki?

Comment (marieroque) [reply helm/prometheus-rules/templates/alerting-rules/loki.rules.yml:76]: You mean with Mimir, the query can match mimir pods ?

Comment (QuentinBisson) [reply helm/prometheus-rules/templates/alerting-rules/loki.rules.yml:76]: I think so yes

**Comment by hervenicol** [comment]
How did you check the new query solves the issue?
I was thinking UTs could be updated to fail with the previous query, and then query would be improved to pass the tests.

**Comment by marieroque** [comment]
> inking UTs could be updated to fail with the previous query, and then query would be improved to pass the te

I tested the query by comparing it on a previous incident on potato.
UT didn't need to be updated as I kept 30 minutes of duration.

**Comment by marieroque** [comment]
I tested previous alert (min_over_time) and new alert (min with for) by adding a new serie in TU.

I could see the previous alert is failing the TUs whereas the new alert passes the TUs.

**Pull request merged**